jsonnet/kube-prometheus/node-exporter/node-exporter.libsonnet

```diff
@@ -69,6 +69,11 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       local containerEnv = container.envType;
 
       local podLabels = $._config.nodeExporter.labels;
+      local selectorLabels = {
```

Comment:
let's move this to the config section and make use of this in the service selector as well

Reply:
The service selector is already picking up these change since it just refers to the daemonset's selector (`$.nodeExporter.daemonset.spec.selector.matchLabels`) but I can definitely make that more explicit. So you'd prefer if both the daemonset and the service used something like `$._config.nodeExporter.selectorLabels` instead?

Reply:
Yeah and have the ServiceMonitor use the same.